### PR TITLE
Migrate to muse hyperparams

### PR DESCRIPTION
**Pull request by raywu0123**
- discriminator  層數更正
- 加新的 google drive 連結
- embedding normalize option 用 argument 控制 
- embedding evaluator min/max bug 修復
- 更新 training script

### CODE REVIEW

**Comment by WangYuNeng** [comment]
我昨天的一個branch也在做這件事ㄟ
那我把我的刪掉好了
